pynest/nest/ll_api.py

```diff
@@ -398,8 +399,17 @@ def init(argv):
         else:
             try:
                 import keyword
```

Comment:
Why not place with other imports on top?

Reply:
See below.